annotation/tests/test_jobs.py

```diff
@@ -0,0 +1,113 @@
+import uuid
+from typing import Set, Tuple
+from unittest.mock import MagicMock
+from uuid import UUID
+
+import pytest
+
+from annotation.errors import FieldConstraintError
+from annotation.jobs.services import check_annotators, update_inner_job_status
+from annotation.models import Category, File, Job, User
+from annotation.schemas import (
+    CategoryTypeSchema,
+    FileStatusEnumSchema,
+    JobStatusEnumSchema,
+    ValidationSchema,
+)
+
+JOB_TENANT = "test"
+JOB_IDS = (1, 2, 3)
+
+
+@pytest.fixture
+def job_annotators():
+    yield (
+        User(user_id="82533770-a99e-4873-8b23-6bbda86b59ae"),
+        User(user_id="ef81a4d0-cc01-447b-9025-a70ed441672d"),
+    )
+
+
+@pytest.fixture
+def categories():
+    yield Category(
+        id="18d3d189e73a4680bfa77ba3fe6ebee5",
+        name="Test",
+        type=CategoryTypeSchema.box,
+    )
+
+
+@pytest.fixture
+def jobs_to_test_progress(
+    job_annotators: Tuple[User], categories: Tuple[Category]
```

Comment:
could you clarify why we yield tuples in those fixtures with a single value? Lets make it a single job, without tuple. Same applies for other fixtures. And here categories fixture is not Tuple, it is Category